src/helpers/paper-util.js

```diff
@@ -352,6 +439,45 @@ const PaperUtil = {
       func(inGridBoundsPoint, ...args)
     }
 
+    this.getTargetPointForKeyboardEvent = (event) => {
+      const focusedPath = this.getFocusedStartingPoint()
+      const ALLOWED_KEYS = ['up', 'down', 'left', 'right']
+      if(focusedPath !== null && focusedPath !== undefined && ALLOWED_KEYS.includes(event.key)) {
+        const gridPoint = this.fromViewCoordinateToGrid(focusedPath.position)
+        const targetPoint = (() => {
+          switch(event.key) {
+            case 'up':
+              return {
+                ...gridPoint,
+                x: gridPoint.x,
+                y:  gridPoint.y + graphSettings.stepY
```

Comment:
Do these operations need a "clamp" so that they can't fall off the grid?

Reply:
The function runs the point through `getClosestInGridPoint` before returning the result. `getClosestInGridPoint` ensures that these are in bound.